examples/webgpu_multiple_rendertargets_with_gpu_readback.html

```diff
@@ -0,0 +1,249 @@
+				renderer.setAnimationLoop( render );
+				document.body.appendChild( renderer.domElement );
+
+				// Create a multi render target with Float buffers
+
+				renderTarget = new THREE.RenderTarget(
+					window.innerWidth * window.devicePixelRatio,
+					window.innerHeight * window.devicePixelRatio,
+					{ count: 2, minFilter: THREE.NearestFilter, magFilter: THREE.NearestFilter }
+				);
+
+				// Name our G-Buffer attachments for debugging
+
+				renderTarget.textures[ 0 ].name = 'diffuse';
+				renderTarget.textures[ 1 ].name = 'normal';
+
+				// Scene setup
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x222222 );
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.1, 50 );
+				camera.position.z = 4;
+
+
+				const loader = new THREE.TextureLoader();
+
+				const diffuse = loader.load( 'textures/hardwood2_diffuse.jpg' );
+				diffuse.colorSpace = THREE.SRGBColorSpace;
+				diffuse.wrapS = THREE.RepeatWrapping;
+				diffuse.wrapT = THREE.RepeatWrapping;
+
+				torus = new THREE.Mesh(
+					new THREE.TorusKnotGeometry( 1, 0.3, 128, 32 ),
+					new WriteGBufferMaterial( diffuse )
+				);
+
+				scene.add( torus );
+
+				
+				// Controls
+
+				new OrbitControls( camera, renderer.domElement );
+
+				window.addEventListener( 'resize', onWindowResize );
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+				const dpr = renderer.getPixelRatio();
+				renderTarget.setSize( window.innerWidth * dpr, window.innerHeight * dpr );
+
+			}
+
+			async function render( time ) {
+
+				torus.rotation.y = ( time / 1000 ) * .4;
+
+				// render scene into target
+				renderer.setRenderTarget( renderTarget );
+				renderer.render( scene, camera );
+
+				// render post FX
+				renderer.setRenderTarget( null );
+
+				if(selector == 1){
+					quadMesh = new QuadMesh( new ReadGBufferMaterial( renderTarget.textures[ 0 ], renderTarget.textures[ 1 ] ) );
+					quadMesh.render( renderer );
+				}
+
+				await readback();
+			}
+
+
+			async function readback(){
+
+				//Be careful with this value! 512 is already big. Reading data back from the GPU is computationally intensive
+				const size = 512;	
+				const renderTarget = new THREE.RenderTarget(size, size, { count: 2 });
+				renderer.setRenderTarget( renderTarget );
+				renderer.render( scene, camera );
+				renderer.setRenderTarget( null );
+
+				const width = renderTarget.width;
+          const height = renderTarget.height;
+				
+				const pixelBuffer0 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 0);	//zero is optional
+				const pixelBuffer1 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 1);
+
+				const pixelBufferTexture0 = new THREE.DataTexture(pixelBuffer0, width, height);
```

Comment:
Please adhere the code style. Lines like this should look like so:
```js
const pixelBufferTexture0 = new THREE.DataTexture( pixelBuffer0, width, height );
```
Your editor should automatically fix such style issues when ESLint is proper supported.